test/llvm/macros/complex-macros.h

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+
+#define ComplexFnLike() \
+  printf("hi\n")
+
+#define ComplexFnLikeArgs(arg) \
+  printf("hi %i\n", arg)
+
+#define MAKE_UNSIGNED(a) \
+  unsigned a
+
+#define ComplexType \
+  MAKE_UNSIGNED(int)
+
+#define ComplexVar \
+  getchar()
```

Comment:
I am not sure why is this calling `getchar()` instead of just being a variable or something either

Reply:
The test is trying to show macros that aren't supported. I wanted to create a pattern that might look syntactically like a variable but isn't.